sources/model/managars/GGHTTPClientManager.m

```diff
@@ -102,7 +102,9 @@ - (NSDictionary * _Nonnull)authenticationHeaders{
     
     NSMutableDictionary *retval = @{@"CLIENT": @"BRINGG SDK iOS",
                                     @"CLIENT-VERSION": SDK_VERSION}.mutableCopy;
-    
+    if (_developerToken!=nil){
```

Comment:
i didn't see who set the value of this. so just making sure its us internally rather than forcing them to right?